// TRY & CATCH
/*
    "Try" significa 'tentar' em português. Ambos os blocos (try e catch) são utilizados
    para tratar erros de nosso código.

    O 'try' é usado para tentarmos rodar um código arriscado ao nosso programa, onde o mesmo
    pode causar erros e interromper a execução de todo o nosso programa. 
    E caso ocorra algum erro dentro do try (por consequência do código arriscado), o `catch`
    é acionado para tratar o erro e evitar que o programa seja encerrado.
    
    Em português, 'catch' significa 'pegar', neste caso, estamos pegando o erro causado pelo
    código que está no 'try' e tratando o mesmo.
    Tratar o erro pode ser um simples ato de fazer o programa não se encerrar, ou até mesmo
    invocar um função que 
*/

// Sintaxe
try{

} catch(erro_recebido){

}

// No bloco try, iremos colocar o nosso código potencialmente perigoso (ou seja, pode gerar erros) ao programa 
// No bloco catch, 